### ISSUE TITLE
enabling notification popups causes crash

### ISSUE THREAD

**Issue by mustaruhtinas** [nullobsi/cantata#1]
Setting ```ShowPopups=true``` triggers the following error and crashes the program.

```
dbus[12040]: Array or variant type requires that type int32 be written, but int64 was written.
The overall signature expected here was '' and we are on byte 163 of that signature.
     D-Bus not built with -rdynamic so unable to print a backtrace 
```
version 3.0.2.r5.ge925ef5-1 running on up-to-date Arch Linux

**Comment by nullobsi** [comment]
hi, how did you install Cantata? Flatpak or a PKGBUILD?

**Comment by mustaruhtinas** [comment]
PKGBUILD, built in a clean chroot.
I do not know whether it is relevant or not, running on wayland without a DE (dunst is used for notifications).

**Comment by nullobsi** [comment]
fixed in db15fc43f9abafcd5f0b3a99c3161f3799c172be

image sizes are represented as `qsizetype` but dbus always expects an `int32` for the image structure for the notification

**Comment by mustaruhtinas** [comment]
Working now, thanks.
